### ISSUE TITLE
ILS Approach Button does not work 

### ISSUE THREAD

**Issue by JRuhestand** [saltysimulations/salty-747#481]
### Version

v0.5.1

### Describe the bug

The Approach button does not work when intersecting the GS. The light does not come on and does not pick up the GS. The localizer button works fine. 

### Expected behaviour

The Approach button to light up and work when pressed.

### Steps to reproduce

1. Create and load FP for ILS landing
2. Select Localizer (works fine)
3. At GS intercept, push the Approach button. Nothing happens

**Comment by JRuhestand** [comment]
Same problem. I tried two different airports. The Approach Mode button is broken.

**Comment by Timberolic** [comment]
The approach mode is **not** broken. I have tested for users multiple times and have shown it functions normally - the worst I have witnessed was the FMA not showing the G/S engaged, but it would still descend accordingly. You need to ensure you are following the proper protocol for FMC setup and NAV RAD page frequency.

**Comment by parzival1925** [comment]
I am also having the same issue, this video shows the same issue I am having and the others are expressing, 
https://www.youtube.com/watch?v=u77rwIgFCsU

**Comment by parzival1925** [comment]
could the issue be incorrect ILS from computer? will test 
https://www.youtube.com/watch?v=vjWg7qM1Jog

**Comment by parzival1925** [comment]
I tested at these airports, CYWG, CYYZ, KIAD, KDEN

at each, if LOC button used, the approach button will not activate, 
if LOC not used, approach seems to activate both LOC and APPROACH, yet LOC is not shown in HUD, and button is no lite. 

the approach floats up and down and can not hold the marker on the line, 

hope this helps

**Comment by JRuhestand** [comment]
> I tested at these airports, CYWG, CYYZ, KIAD, KDEN
> 
> at each, if LOC button used, the approach button will not activate, if LOC not used, approach seems to activate both LOC and APPROACH, yet LOC is not shown in HUD, and button is no lite.
> 
> the approach floats up and down and can not hold the marker on the line,
> 
> hope this helps

Parzival1925, 

You are 100% correct. Those are the exact same issues I experienced regardless of airports. Seems to function more like the B787 than the B747. It is still an issue despite Timberolic saying the problem doesn't exist and it's user error.

**Comment by parzival1925** [comment]
Yes, I tested yesterday those airports with the salty 747, Today I tried a holding pattern, the holding pattern worked perfect, it was about .25 to .5 km off the mark yet did it smooth, even coming into it to fast and pitched down hard, 

At LAX, the same issue, yet not the exact same, LOC worked, and the APP worked, yet the APP button did not light up and did not register on display, this time I did enter in the step elevations and speeds, kept a nice pitch and stayed dead center of both marks all the way in, I noticed the VNAV sometime needs 2 or 3 presses before it connects, sometimes have to press VS then VNAV a few times to get the VNAV to work after using heading select.

the speed control from computer sometimes works, and others does not, this is consistent with all flights,

**Comment by Timberolic** [comment]
I'm going to test two/three approaches and prove the ILS system is fine, both LOC -> APP and straight into APP - speed control meaning what? FMC speed (not implemented) or MCP Speed that you set on the autopilot panel?

**Comment by parzival1925** [comment]
speed set into computer, the one on the panel works fine, 

try one of these 
CYWG, CYYZ, KIAD, KDEN

**Comment by Timberolic** [comment]
Will do - As stated before, VNAV speeds in the FMC are not implemented yet

**Comment by parzival1925** [comment]
sometime, it does take those speeds, and I have had to reset throttle control to set my own,

**Comment by parzival1925** [comment]
it is m favorite plane, anything I can do to help let me know :)

**Comment by Timberolic** [comment]
It will only take the speed you set on the VNAV Page - but if you try to set it on LEGS it won't work. If it overspeeds what you inputted on the VNAV page, you need drag ie. Spoilers

**Comment by parzival1925** [comment]
do you have a guide for that function page?

**Comment by Timberolic** [comment]
Not much to it - set your desired SPD on the SPD tab of CLB page, CRZ page and DES page. Though I don't use it for DES because most STARs and their speed restrictions change over time

**Comment by e8vww** [comment]
its working for me.. just tested at TJSJ. If you are new to this a/c it is very easy to approach too fast and the a/p can only do so much. part of the problem is the lack of basic instructions/ documentation. i had to look at a bunch of youtube videos and piece things together.
